Products/ZenPackAdapter/utils/tales.py

```diff
@@ -49,11 +49,8 @@ def talesEval(express, context, extra=None, skipfails=False):
         msg = "Error when processing tales expression %s on context %s : Exception Class %s Message: %s" % (express,
                                                                                                             context,
                                                                                                             type(e), e)
-        if skipfails:
-            res = express
-            log.debug(msg)
-        else:
-            raise InvalidTalesException(msg)
+        res = express
+        log.warn(msg)
```

Comment:
Is this still needed?  It looks like you're catching the InvalidTalesExpression and logging it elsewhere.

Reply:
I couldn't get zpa or the EMC zenpack to load correctly until i removed the "raise InvalideTalesException(msg)"; the exception was causing the zenpack to fail loading.  Hence, I updated this segment to operate similar to how res_manager operates; returning the expression unmodified.

From there, I never really re-visited this since ZPA has been running smooth for me since (so I could work on events.)